### PR TITLE
Fix issue fetch associated experiment runs for commit

### PR DESCRIPTION
**Pull request by anandjakhaniya**
**Jira ticket:** https://vertaai.atlassian.net/browse/VR-4475

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java [hunk 1]
```diff
@@ -1070,23 +1070,53 @@ public WorkspaceDTO getWorkspaceDTOByWorkspaceName(
...
+                authService.getUserInfo(workspaceName, ModelDBConstants.UserIdentifier.USER_NAME));
+        workspaceDTO.setWorkspaceId(collaboratorUser.getId());
         workspaceDTO.setWorkspaceType(WorkspaceType.USER);
+        workspaceDTO.setWorkspaceName(workspaceName);
       }
     }
     return workspaceDTO;
   }
 
+  @Override
```
Comment (ravishetye) [inline backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java:1096]:
```suggestion
  /**
  * Given the workspace id and type, returns WorkspaceDTO which has the id, name and type fo the workspace.
  **/
  @Override
```

Comment (ravishetye) [reply backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java:1096]: pull and mvn package for it to be formatted correctly

**Review by ravishetye — APPROVED** [review]
added a comment ass suggestion

File: backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java:1096]: Added it and formatted

**Pull request merged**